notebooks/ImvizExample.ipynb

```diff
@@ -577,7 +577,8 @@
     "                         SkyCoord('00h24m01.57s -71d53m17.77s'),\n",
     "                         SkyCoord('00h24m11.70s -71d52m29.21s'),\n",
     "                         SkyCoord('00h24m22.29s -71d53m28.04s')]})\n",
-    "viewer.add_markers(t_sky, use_skycoord=True, marker_name='my_sky')"
+    "viewer.add_markers(t_sky, use_skycoord=True, marker_name='my_sky')\n",
+    "t_sky"
```

Comment:
@orifox , is this intentional or leftover debugging? Doesn't hurt to leave it in, I guess.